src/briefcase/platforms/web/static.py

```diff
@@ -96,33 +100,365 @@ def _trim_file(self, path, sentinel):
+                                f" * {package_key}::{rel_inside}\n"
+                                " *******************************************************/\n\n"
+                            ) + css_text
+
+                            # Add CSS content to briefcase.css insert slot
+                            target = "static/css/briefcase.css"
+                            insert = "CSS"
+                            pkg_map = inserts.setdefault(target, {}).setdefault(
+                                insert, {}
+                            )
+                            # Append if package already has content for this slot
+                            if package_key in pkg_map and pkg_map[package_key]:
+                                pkg_map[package_key] += "\n" + css_payload
+                            else:
+                                pkg_map[package_key] = css_payload
+
+    def extract_backend_config(self, wheels):
+        """Processes multiple wheels to gather a config.toml and a base pyscript.toml
+        file.
+
+        :param wheels: A list of wheel files to be scanned.
+        """
+        config_package = None
+        config_package_list = []
+        config_filename = None
+        pyscript_version = "2024.11.1"
+        pyscript_config = None
+
+        # Find packages containing a config.toml file.
+        for wheelfile in wheels:
+            with ZipFile(wheelfile) as wheel:
+                for filename in wheel.namelist():
+                    path = Path(filename)
+                    if (
+                        len(path.parts) == 3
+                        and path.parts[1] == "deploy"
+                        and path.name == "config.toml"
+                    ):
+                        self.console.info(f"    Found {filename}")
+                        config_package_list.append(wheelfile)
+                        config_filename = filename
+
+        # Return a blank pyscript config if no configuration file is found.
+        if len(config_package_list) == 0:
+            pyscript_config = {}
+        # Raise an error if more than one configuration file is supplied.
+        elif len(config_package_list) > 1:
+            raise BriefcaseConfigError(
+                f"""Only 1 backend configuration file can be supplied.
+                Initial config.toml found in package: {config_package}
+                Duplicate config.toml found in package: {wheel.filename}"""
+            )
+        # Gather a backend configuration file from the package.
+        # For now, is a pyscript.toml as no other backend is currently supported.
+        else:
+            with ZipFile(config_package_list[0]) as wheel:
+                # Check which backend type is used.
+                with wheel.open(config_filename) as config_file:
+                    config_data = tomllib.load(config_file)
+
+                    if "backend" in config_data:
+                        backend = config_data.get("backend")
```

Comment:
This reads as a `try:except:` - or, at least, a series of "if bad, exception; if bad for other reason; exception" before doing the actual processing of the "good" case.